Source/WebInspectorUI/UserInterface/Views/SpreadsheetStyleProperty.js

```diff
@@ -368,6 +368,13 @@ WI.SpreadsheetStyleProperty = class SpreadsheetStyleProperty extends WI.Object
         textField.value = isEditingName ? this._property.name : this._property.rawValue;
     }
 
+    spreadsheetTextFieldInitialCompletionIndex(textField, completions)
+    {
+        if (textField === this._nameTextField && WI.settings.experimentalCSSSortPropertyNameAutocompletionByUsage.value)
+            return completions.minIndex(WI.CSSProperty.sortByPropertyNameUsageCount);
```

Comment:
I'm tempted to add some logic here (or to `minIndex`)  that ensures the value at `minIndex` is greater by some minimum than the next `minIndex` so that if we had a situation where `width` is seen `101` times and `will-change` is seen `100` times we don't end up flip flopping back and forth between which is initially suggested (i.e. if `will-change` happens to be seen twice more, then it'd become the default with `102` times).

My concern with this though is that then things like `padding` and `padding-top` which are probably used with similar frequency would end up "clashing", resulting in neither being initially suggested, which is arguably worse.

@patrickangle any thoughts?

Reply:
My guess is that in practice this is unlikely to be an issue, and if it is it means folks are truly using both almost equally, at which point they are probably used to having to type more than a character or two to get the completion they want.

Your mention of `padding` v. `padding-top` is perhaps more interesting because IMO we should always provide the shorthand in that case until the entry is specific enough to get the longhand (it's quite easy to just `p-` with the fuzzy matching support, at which point we would surface `padding-top` instead of say `padding-bottom` if the former is used more than the later).